css/sytles.css

```diff
@@ -0,0 +1,58 @@
+div.container {
+  width: 736px;
+  margin: 0 auto;
+}
+
+h1 {
+  text-align: center;
+  padding: 20px;
+  background-color: yellow;
+  border-style: dashed;
+  border-width: 2px;
+  border-color: green;
+  margin-bottom: 0px;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+nav {
+  text-align: right;
+  padding: 10px;
+  background-color: yellow;
+  border: 1px;
```

Comment:
@ermajinkles This could have written as 
` nav { 
    border: 1px dotted green;
}
`